src/site/content/en/blog/coop-coep/index.md

```diff
@@ -0,0 +1,296 @@
+is required for step four, enable COEP.
+
+Here is what you need to do depending on the nature of the resource:
+
+* If the resource is expected to be loaded **only from the same origin**, set 
+  the `Cross-Origin-Resource-Policy: same-origin` header.
+* If the resource is expected to be loaded **only from the same site but cross 
+  origin**, set the `Cross-Origin-Resource-Policy: same-site` header.
+* If the resource is **loaded from cross origin(s) under your control**, set the 
+  `Cross-Origin-Resource-Policy: cross-origin` header if possible. 
+* For cross origin resources that you have no control over:
+    * Use the `crossorigin` attribute in the loading HTML tag if the resource is
+      served with CORS.
+    * Ask the owner of the resource to support either CORS or CORP.
+* For iframes, use CORP and COEP headers as follows: 
+  `Cross-Origin-Resource-Policy: same-origin` and `Cross-Origin-Embedder-Policy: 
+  require-corp`.
+
+{% Aside 'key-term' %}
+It's important that you understand the difference between "same-site" and 
+"same-origin". Learn about the difference at "[Understanding same-site and 
+same-origin](/same-site-same-origin)".
+{% endAside %}
+
+#### 3. Use the COEP Report-Only HTTP header to assess embedded resources
+
+Before fully enabling COEP, you can do a dry run by using the
+`Cross-Origin-Embedder-Policy-Report-Only` header to examine whether the policy
+actually works. You will receive reports without blocking embedded content.
+Recursively apply this to all documents. for information on Report-Only HTTP
+header, see [Observe issues using the Reporting
+API](#observe-issues-using-the-reporting-api).
+
+#### 4. Enable COEP
+
+Once you've confirmed that everything works, and that all resources can be 
+successfully loaded, apply the `Cross-Origin-Embedder-Policy: require-corp` HTTP 
+header to all documents including those that are embedded via iframes.
+
+{% Aside 'codelab' %}
+[See the impact of different COEP / CORP 
+parameters](https://first-party-test.glitch.me/coep).
+{% endAside %}
+
+### Determine whether isolation succeeded with `self.crossOriginIsolated`
+
+`self.crossOriginIsolated` returns `true` when the web page is in a 
+cross-origin isolated state and all resources and windows are isolated within 
+the same browsing context group. You can use this API to determine whether you 
+have successfully isolated the browsing context group and gained access to 
+powerful features like `performance.measureMemory`.
+
+{% Aside 'caution' %}
+[`self.crossOriginIsolated`](https://www.chromestatus.com/feature/5953286387531776)
+is still under development and not available yet in Chrome as of version 83.
+{% endAside %}
+
+### Debug issues using Chrome DevTools
+
+For resources that are rendered on the screen such as images, it's fairly easy 
+to detect COEP issues because the request will be blocked and the user 
+experience will be visually broken. However, for resources that don't 
```

Comment:
```suggestion
to detect COEP issues because the request will be blocked and the page will
indicate a missing image. However, for resources that don't 
```